### ISSUE TITLE
matrixElements is missed

### ISSUE THREAD

**Issue by kopitoshka** [wout/svg.js#373]
matrixElements is missed in SVG.regex = { declaration.

It causes issues in IE11. Could you please add that line there. I would suggest a line like this:

,  matrixElements: /\s/g

**Comment by Fuzzyma** [comment]
What kind of issues? How can it do anything? SVG.regex is just an object with all used regex in the svg.js library. matrixElements is nowhere used in the lib so why I should define it there?

**Comment by kopitoshka** [comment]
I am sorry, it was my mistake. While I examined the library with draggable plug-in I added it myself to the code. 

But I had to add that constant anyway to SVG.regex object because a line below may include spaces and commas, not just commas. With a default svg file my test draggable object did not work properly.

I replaced this line:

```javascript
return [kv[0], kv[1].split(',').map(function(str){ return parseFloat(str) })]
```
with this one:

```javascript
return [kv[0], kv[1].split(SVG.regex.matrixElements).map(function(str){ return parseFloat(str) })]
```

and added a new SVG.regex constant:

```javascript
, matrixElements: /\s|,/g
```

I hope it will help. Anyway, feel free to close the ticket.
